src/frontend-samples/hypermodeling-sample/HyperModelingUI.tsx

```diff
@@ -0,0 +1,131 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "common/samples-common.scss";
+import { ControlPane } from "common/ControlPane/ControlPane";
+import { SandboxViewport } from "common/SandboxViewport/SandboxViewport";
+import * as React from "react";
+import { assert, Id64String } from "@bentley/bentleyjs-core";
+import { IModelApp, ScreenViewport, ViewState } from "@bentley/imodeljs-frontend";
+import { SectionMarker } from "@bentley/hypermodeling-frontend";
```

Comment:
In problems pane: Cannot find module '@bentley/hypermodeling-frontend' or its corresponding type declarations.

See [Stefan's comments](https://github.com/imodeljs/frontend-sample-showcase/pull/160#issuecomment-792984872)

Reply:
Fixed (this should be documented somewhere?)

Reply:
I believe there is a bit of info in Contributing.md, but could be clarified.